rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_publisher.hpp

```diff
@@ -130,7 +124,29 @@ class LifecyclePublisher : public LifecyclePublisherInterface,
   }
 
 private:
+  /**
+   * @brief Helper function that logs a message saying that publisher can't publish
+   * because it's not enabled.
+   */
```

Comment:
```suggestion
  /// Helper function that logs a message saying that publisher can't publish because it's not enabled.
```

Reply:
We don't use `@` but instead use `\` in our doxygen blocks, and we prefer an implicit "brief" avoiding the command.

Reply:
By the way, my suggestion is too long, so either shorten the message or do multi-line like this:

```suggestion
   /// Helper function that logs a message saying that publisher can't publish
   /// because it's not enabled.
   /**
    * Anything else you have to say/document.
    */
```